how-we-process-data-five-times-more-efficiently-using-a-scale-out-mysql-alternative.md

```diff
@@ -0,0 +1,138 @@
+---
+title: 'How We Process Data Five Times More Efficiently Using a Scale-Out MySQL Alternative'
+author: ['Yu Liu']
+date: 2020-11-19
+summary: A senior architect at China Telecom Bestpay shared how TiDB, a MySQL-compatible, distributed SQL database, boosted the performance of their application system by 3–5 times.
+tags: ['MySQL compatibility', 'Scalability']
+url: /success-stories/how-we-process-data-five-times-more-efficiently-using-a-scale-out-mysql-alternative/
+customer: China Telecom
```

Comment:
Should I write "China Telecom" or "China Telecom Bestpay"? @CaitinChen

Reply:
After a discussion with @lilin90, we suggest "China Telecom Bestpay."

Reply:
```suggestion
customer: China Telecom Bestpay
```